suites/pacific/cephadm/tier_0_cephadm.yaml

```diff
@@ -497,6 +498,15 @@ tests:
               - node2
       destroy-cluster: false
       abort-on-fail: true
+  - test:
```

Comment:
As discussed, Can we take this in with `--grafana-image` cli option and update whenever needed for test-suites which have `-skip-monitoring-stack` options.